pkg/chunk/aws/dynamodb_table_client.go

```diff
@@ -189,6 +189,24 @@ func (d dynamoTableClient) CreateTable(ctx context.Context, desc chunk.TableDesc
 	return nil
 }
 
+func (d dynamoTableClient) DeleteTable(ctx context.Context, name string) error {
+	if err := d.backoffAndRetry(ctx, func(ctx context.Context) error {
```

Comment:
Seeing this prompted me to file #1152. (just a comment; no action required in this PR)